CoreAlgorithmPlugins/test/unit/src/au/gov/asd/tac/constellation/plugins/algorithms/sna/centrality/PagerankCentralityPluginNGTest.java

```diff
@@ -84,16 +84,16 @@ public void testPagerank() throws Exception {
         final PluginParameters parameters = instance.createParameters();
         parameters.setBooleanValue(PagerankCentralityPlugin.TREAT_UNDIRECTED_BIDIRECTIONAL_PARAMETER_ID, true);
         parameters.setFloatValue(PagerankCentralityPlugin.DAMPING_FACTOR_PARAMETER_ID, 0.85f);
-        parameters.setIntegerValue(PagerankCentralityPlugin.ITERATIONS_PARAMETER_ID, 100);
+        parameters.setIntegerValue(PagerankCentralityPlugin.ITERATIONS_PARAMETER_ID, 2);
         parameters.setFloatValue(PagerankCentralityPlugin.EPSILON_PARAMETER_ID, 1E-8f);
         parameters.setBooleanValue(PagerankCentralityPlugin.NORMALISE_AVAILABLE_PARAMETER_ID, false);
         PluginExecution.withPlugin(instance).withParameters(parameters).executeNow(graph);
 
-        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId0), 0.08510862f);
-        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId1), 0.15745096f);
-        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId2), 0.15202528f);
-        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId3), 0.28124678f);
-        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId4), 0.32416838f);
+        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId0), 0.0725f);
+        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId1), 0.134125f);
+        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId2), 0.1755625f);
+        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId3), 0.3094375f);
+        assertEquals(graph.getFloatValue(vertexPagerankAttribute, vxId4), 0.308375f);
```

Comment:
@Nova-2119 @cygnus-x-1 Previously `vxId4` was last but now its `vxId3`. Does that mean the page rank was wrong before or wrong now?

Reply:
It was wrong before, all the values were slightly out because when there was a sink it included itself in the count of how many nodes it was distributing its pagerank too when it shouldn't have (it used the total number of nodes rather than number of nodes-1).
This has changed all the values a little.

Reply:
ok thank you @Nova-2119